app/views/pages/index.html.haml

```diff
@@ -1 +1,34 @@
-= render 'layouts/nav'
+= render :partial => 'layouts/nav'
+
+.container.mt-5
+
+  %button.btn.btn-primary.mb-3(type="button") Share your Journey
+
+  .card
+    .card-body
+      .media
+        = image_tag "https://picsum.photos/130/170", class: "align-self-start mr-3"
```

Comment:
Consider placing the image in your assets folder and using `image_pack_tag` to retrieve it, instead of linking to an external site.

Reply:
Currently the images being displayed here are coming from an image randomizer api so I'll keep it temporarily then update it later on.